core/controllers/base.py

```diff
@@ -605,7 +663,9 @@ def render_template(self, filepath, iframe_restriction='DENY'):
 
         self.response.write(load_template(filepath))
 
-    def _render_exception_json_or_html(self, return_type, values):
+    def _render_exception_json_or_html(
+        self, return_type: Optional[str], values: ResponseValueDict
```

Comment:
Shouldn't the return_type be non-optional?

Reply:
Yeah, you're right. Fixed!